vmm/src/acpi.rs

```diff
@@ -402,6 +402,68 @@ fn create_spcr_table(base_address: u64, gsi: u32) -> Sdt {
     spcr
 }
 
+#[cfg(target_arch = "aarch64")]
+fn create_dbg2_table(base_address: u64) -> Sdt {
+    let ns = b"_SB_.COM1";
+    let ddi_off = 44usize;
+    let ddi_len: u16 = 22 + 12 + 4 + ns.len() as u16 + 1;
```

Comment:
The adding of 3 numbers doesn't provide much information itself.
How about adding some comments about the calculation `22 + 12 + 4`, or simply writing `38`?

Reply:
Addressed both suggestions:

- Added comments to the numbers. The numbers are just to illustrate on how the dynamic part of the debug device info struct is taken into account (the members after 22 bytes offset).
- Written out the variable names. 

Thanks